snapshot-labs/snapshot-sidekick
# feat: export digital ocean metrics

- Export data from DigitalOcean through /metrics 
- Add authentication to the /metrics endpoint

How to test:

- Set you digitalOcean api token key in the .env (`DIGITAL_OCEAN_TOKEN`)
- Go to `http://localhost:3005/metrics`
- It should show metrics starting by `do_`
